compiler/damlc/tests/daml-test-files/ImplementsNonInterface.daml

```diff
@@ -0,0 +1,18 @@
+-- Copyright (c) 2022, Digital Asset (Switzerland) GmbH and/or its affiliates.
+-- All rights reserved.
+
+-- Test the serializability checker for Daml-LF.
+-- We use the template typeclass and instances directly as otherwise the error
+-- is caught prior due to missing Eq and Show instances.
+
+-- @SINCE-LF-FEATURE DAML_INTERFACE
+-- @ERROR range=17:5-17:20; cannot implement 'Bool' because it is not an interface
```

Comment:
ohno, line numbers are 1-indexed; this is failing with the message below, see [CI](https://dev.azure.com/digitalasset/daml/_build/results?buildId=115416&view=logs&j=0da5d1d9-276d-5173-c4c4-9d4d4ed14fdb&t=7b720ed4-ea9c-5605-45fe-aca5cec61df4&l=9060)

```
      Range:    18:5-18:20
      Source:   Core to Daml-LF
      Severity: DsError
      Message:  cannot implement 'Bool' because it is not an interface
      Could not find matching diagnostics:
      -- @ERROR range=17:5-17:20; cannot implement 'Bool' because it is not an interface
```


```suggestion
-- @ERROR range=18:5-18:20; cannot implement 'Bool' because it is not an interface
```

Reply:
I'm confident this would have succeeded if the line numbers had been right, so you may merge the DA-GHC PR, point daml to the merge commit, and fix the expected line numbers, without having to test against the https://github.com/digital-asset/ghc/pull/125 branch again